perun-cli/getSpecificUsersByUser

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Text::ASCIITable;
+use Perun::Agent;
+use Perun::Common qw(printMessage tableToPrint);
+
+sub help {
+	return qq{
+	Lists Specific users assigned to User. User id is required.
+	--------------------------------------
+	Available options:
+	--userId       | -u user id
+	--batch        | -b batch
+	--help         | -h prints this help
+
+	};
+}
+
+my ($userId, $ownerId, $batch);
+GetOptions ("help|h"  => sub {
+		print help();
+		exit 0;
+	}, "batch|b"        => \$batch,
+	"userId|u=i"        => \$userId ) || die help();
+
+# Check options
+unless (defined($userId)) { die "ERROR: userId is required \n";}
+
+my $agent = Perun::Agent->new();
+my $usersAgent = $agent->getUsersAgent;
+
+my @users=$usersAgent->getSpecificUsersByUser( user => $userId );
+
+#output
+my $table = Text::ASCIITable->new({ reportErrors => 0, utf8 => 0 });
```

Comment:
Please use common implementation of printTable() on User object. It contains all necessary columns. See ``listOfUsers`` tool.